processor/configuration/src/test/java/com/redhat/service/smartevents/processor/validators/DefaultGatewayValidatorTest.java

```diff
@@ -183,6 +184,24 @@ void testKafkaTopicAction() {
         assertValidationIsValid(validKafkaTopicAction);
     }
 
+    @Test
+    void testAzureEventHubAction() {
+        Map<String, String> invalidParams = new HashMap<>();
+        invalidParams.put(AzureEventHubsAction.AZURE_SHARED_ACCESS_KEY_PARAM, "example");
```

Comment:
```suggestion
        Map<String, String> invalidParams = java.util.Collections.singletonMap(AzureEventHubsAction.AZURE_SHARED_ACCESS_KEY_PARAM, "example");
```

Reply:
I would refactor this in another PR if it's ok by you